clients/drcachesim/CMakeLists.txt

```diff
@@ -583,7 +586,7 @@ add_custom_command(
   COMMAND ${CMAKE_COMMAND}
   ARGS -D src=${srcdoc}
        -D dst=${gendoc}
-       -D prog=${ops_path}
+       -D prog=$<TARGET_FILE:drcachesim_ops>${location_suffix}
```

Comment:
This doesn't look right: we don't want "_Debug" on the end of the path.  How do we specify the config type to this $<TARGET_FILE:xxx> syntax?

Same thing below as well.

Reply:
I don't see a reply here: what does this turn into for VS?

Reply:
These and other comments below were hidden, didn't see, will address.

Reply:
Do we understand how they were hidden in your view?  Did you commit changes to the same lines of code where the comments were before looking at all of the comments?  I think that is when Github marks as "outdated" or sthg.  Is that what you mean by "hidden", still there but collapsed?

Reply:
No problem with GitHub, they were just collapsed with a "hidden" link and I thought they were resolved (or otherwise not be collapsed).

Reply:
The old ops_path also contained ${location_suffix}
Is there a AppVeyor test for building documentation under Windows?

Reply:
The old ops_path did this:

```
get_target_property(ops_path drcachesim_ops LOCATION${location_suffix})
```

That turns into the LOCATION_DEBUG or LOCATION_RELWITHDEBINFO property.  I.e., ${location_suffix} is on the property name, not on the path.

Reply:
I didn't even know LOCATION_DEBUG etc. exists. I'm able to build Win docs now, checking this out WIP.

Reply:
I verified on Windows that TARGET_FILE seems to reflect the correct name.

Reply:
So `$<TARGET_FILE:drcachesim_ops>_DEBUG` is the correct syntax?  It seems weird: I'd expect `$<TARGET_FILE_DEBUG:drcachesim_ops>`.

Reply:
No it's like you said, the branch here was updated to remove the suffix.

Reply:
Who adds the configuration-specific subdir for multi-configuration generators like VS or XCode, then, where a single CMake config can build either Debug or Relwithdebinfo and uses a subdir in the build dir?  Is that built in to generator expressions somehow?  DR collapses them to one, but still had to use the LOCATION_xxxx b/c plain LOCATION was empty IIRC for VS generators -- are we sure these TARGET_FILE generator expressions do the right thing?  Can you point at any docs on this?  I couldn't find good info on a quick search.

Reply:
Yes so I can't answer this yet as well. What I did was to check whether we're building the same way (i.e. prog has the same value) in Windows. I am assuming the generator expression provides the final correct path dependent on what you're building. I checked RelWithDebInfo and Debug and both times the expression had the right path. I'll see if I can find some docs

Reply:
Mailing list replied that TARGET_FILE generator expression does give the per-config location for multi-config generators.